libtiledbsoma/src/soma/soma_array.h

```diff
@@ -515,9 +522,9 @@ class SOMAArray : public SOMAObject {
     /**
      * @brief Get the Arrow schema of the array.
      *
-     * @return std::unique_ptr<ArrowSchema> Schema
+     * @return std::shared_ptr<std::unique_ptr<ArrowSchema>> Schema
```

Comment:
shared_ptr of unique_ptr of ArrowSchema ?

Reply:
Thanks for catching this!